<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchProviderDlgUI</class>
 <widget class="QWidget" name="SearchProviderDlgUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>143</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="lbName">
     <property name="whatsThis">
      <string>Enter the human-readable name of the search provider here.</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Shortcut &amp;name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>leName</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="lbQuery">
     <property name="whatsThis">
      <string>&lt;qt&gt;
Enter the URI that is used to perform a search on the search engine here.&lt;br/&gt;The whole text to be searched for can be specified as \{@} or \{0}.&lt;br/&gt;
Recommended is \{@}, since it removes all query variables (name=value) from the resulting string, whereas \{0} will be substituted with the unmodified query string.&lt;br/&gt;You can use \{1} ... \{n} to specify certain words from the query and \{name} to specify a value given by 'name=value' in the user query.&lt;br/&gt;In addition it is possible to specify multiple references (names, numbers and strings) at once (\{name1,name2,...,&quot;string&quot;}).&lt;br/&gt;The first matching value (from the left) will be used as the substitution value for the resulting URI.&lt;br/&gt;A quoted string can be used as the default value if nothing matches from the left of the reference list.
&lt;/qt&gt;</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Shortcut &amp;URL:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>leQuery</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="leQuery">
     <property name="toolTip">
      <string>&lt;qt&gt;
Enter the URI that is used to perform a search on the search engine here.&lt;br/&gt;The whole text to be searched for can be specified as \{@} or \{0}.&lt;br/&gt;
Recommended is \{@}, since it removes all query variables (name=value) from the resulting string, whereas \{0} will be substituted with the unmodified query string.&lt;br/&gt;You can use \{1} ... \{n} to specify certain words from the query and \{name} to specify a value given by 'name=value' in the user query.&lt;br/&gt;In addition it is possible to specify multiple references (names, numbers and strings) at once (\{name1,name2,...,&quot;string&quot;}).&lt;br/&gt;The first matching value (from the left) will be used as the substitution value for the resulting URI.&lt;br/&gt;A quoted string can be used as the default value if nothing matches from the left of the reference list.
&lt;/qt&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;qt&gt;
Enter the URI that is used to perform a search on the search engine here.&lt;br/&gt;The whole text to be searched for can be specified as \{@} or \{0}.&lt;br/&gt;
Recommended is \{@}, since it removes all query variables (name=value) from the resulting string, whereas \{0} will be substituted with the unmodified query string.&lt;br/&gt;You can use \{1} ... \{n} to specify certain words from the query and \{name} to specify a value given by 'name=value' in the user query.&lt;br/&gt;In addition it is possible to specify multiple references (names, numbers and strings) at once (\{name1,name2,...,&quot;string&quot;}).&lt;br/&gt;The first matching value (from the left) will be used as the substitution value for the resulting URI.&lt;br/&gt;A quoted string can be used as the default value if nothing matches from the left of the reference list.
&lt;/qt&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QComboBox" name="cbCharset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>Select the character set that will be used to encode your search query.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="leShortcut">
     <property name="toolTip">
      <string>&lt;qt&gt;
The shortcuts entered here can be used as a pseudo-URI scheme in KDE. For example, the shortcut &lt;b&gt;av&lt;/b&gt; can be used as in &lt;b&gt;av&lt;/b&gt;:&lt;b&gt;my search&lt;/b&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;qt&gt;
The shortcuts entered here can be used as a pseudo-URI scheme in KDE. For example, the shortcut &lt;b&gt;av&lt;/b&gt; can be used as in &lt;b&gt;av&lt;/b&gt;:&lt;b&gt;my search&lt;/b&gt;
&lt;/qt&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="leName">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the human-readable name of the web shortcut here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the human-readable name of the web shortcut here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="lbShortcut">
     <property name="whatsThis">
      <string>&lt;qt&gt;
The shortcuts entered here can be used as a pseudo-URI scheme in KDE. For example, the shortcut &lt;b&gt;av&lt;/b&gt; can be used as in &lt;b&gt;av&lt;/b&gt;:&lt;b&gt;my search&lt;/b&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="text">
      <string>&amp;Shortcuts:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>leShortcut</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="lbCharset">
     <property name="whatsThis">
      <string>Select the character set that will be used to encode your search query</string>
     </property>
     <property name="text">
      <string>&amp;Charset:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>cbCharset</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLabel" name="noteLabel">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pbPaste">
     <property name="toolTip">
      <string>Insert query placeholder</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset theme="edit-paste"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leQuery</tabstop>
  <tabstop>leShortcut</tabstop>
  <tabstop>cbCharset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
